docs/content/blog/ignoring-errors.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "Ignoring Mixin Errors and Idempotent Actions"
+description: "Porter now supports ignoring the errors from a mixin. The az mixin takes advantage of this new feature to manage resource groups."
+date: "2022-01-07"
+authorname: "Carolyn Van Slyck"
+author: "@carolynvs"
+authorlink: "https://twitter.com/carolynvs"
+authorimage: "https://github.com/carolynvs.png"
+tags: ["mixins"]
+summary: |
+    Porter now supports ignoring the errors from a mixin. The az mixin takes advantage of this new feature to manage resource groups.
+---
+
+When is an error not really an error?
+Questions like this come up regularly when automating deployments.
+For example, when I repeat a script that creates a resource, I don't want it to error out because the resource already exists.
+A lot of command-line tools weren't designed in a way that works well with bundles.
+An installation could fail halfway through and need to be repeated.
+This is why we encourage bundle authors to use custom mixins, instead of just the exec mixin, because mixins are designed to work with bundles and give you **idemponent** behavior.
+Mixins should do what's necessary to match the desired state in the bundle and handle recoverable errors.
+
+Since we can't have a custom mixin for every tool and situation though, the exec mixin just got smarter.
```

Comment:
mix of pronouns in this post. 

based on the previous sentence about encouraging bundle authors to leverage custom mixins maybe instead something here would be 

At times it may not be possible to implement and maintain custom mixins for all of your tools and situations, so we've made the exec mixin smarter.